graphics/jpegoptim/Portfile

```diff
@@ -8,7 +8,7 @@ github.tarball_from archive
 revision            0
 categories          graphics
 license             GPL-2+
-maintainers         casa-z.org:jingoro
+maintainers         casa-z.org:jingoro openmaintainer
```

Comment:
please send an e-mail to confirm; the current maintainer will get no notification of this. Or, alternatively, file a "port abandoned" Trac ticket.